cityofaustin/atd-moped
# Removes workgroup column from users table

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/10020

## Testing
**URL to test:** Local or Netlify


**Steps to test:**
You can test the app is working normally by visiting these pages:
1. Staff view - observe that the "workgroup" column is missin
2. Profile view
3. Project list with team column enabled
4. Team tab of project summary

You will not be able to test the add or edit staff forms. If you'd like to test the moped API changes i am happy to help you get that running. I did test my changes successfully. 

---
#### Ship list
